interpreter/llvm/Module.mk

```diff
@@ -193,16 +193,17 @@ $(LLVMDEPO): $(LLVMDEPS)
 		-DLLVM_INCLUDE_EXAMPLES=OFF \
 		-DLLVM_FORCE_USE_OLD_TOOLCHAIN=ON \
 		-DLLVM_ENABLE_THREADS=OFF \
+		-DLLVM_ENABLE_ZLIB=OFF \
```

Comment:
That would make our PCH ~15% bigger. We should have zlib enabled for LLVM because it will compress the embedded textual header files.

Reply:
Ok, could you then take a look at another way to fix this problem in LLVM itself? Since it ends up inside a `#define`, I do not see an easy fix (i.e., linking against the imported target wouldn't work).

Reply:
We've been working on that problem for a while and we failed to find a reasonable solution. I am working on a proposal which ideally resolves that but it is not a trivial fix.

Reply:
The problem is the commit linked in the PR description. That commit uses a very hacky way of adding linking options. The equally hacky solution would be to check if the option is an existing file name and in that case not add the `-l` in front.